### PR TITLE
Implement New toolkit version csharp samples

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**
**Sample List -** 

**1. bot-archive-groupchat-messages/csharp
2. msgext-search-sso-config/csharp
3. graph-appcatalog-lifecycle/csharp
4. meetings-context-app/csharp
5. bot-sharepoint-list/csharp**

### CODE REVIEW

File: samples/bot-sharepoint-list/csharp/TeamsApp/launchSettings.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿{
+  "profiles": {
+    // Debug project within Teams
+    "Microsoft Teams (browser)": {
+      "commandName": "Project",
+      "launchUrl": "https://teams.microsoft.com/l/app/%TEAMSAPPID%?installAppPackage=true&webjoin=true&appTenantId=%TENANTID%&login_hint=%USERNAME%"
```
Comment (Alive-Fish) [inline samples/bot-sharepoint-list/csharp/TeamsApp/launchSettings.json:6]: The launchUrl seems wrong. Please use `${{TEAMS_APP_ID}}` replacing `%TEAMSAPPID%`, also, check other variables and `launchUrl`.

**Review by Alive-Fish — APPROVED** [review]
Only find the above obvious issue. Please check the previous comment.

File: samples/bot-sharepoint-list/csharp/TeamsApp/launchSettings.json (hunk 1 above)
Comment (Harikrishnan-MSFT) [reply samples/bot-sharepoint-list/csharp/TeamsApp/launchSettings.json:6]: Fixed

**Review by Alive-Fish — APPROVED** [review]

**Review by Mohammed-MSFT — APPROVED** [review]

**Comment by Harikrishnan-MSFT** [comment]
Build Pipeline - 
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/11327628738/workflow

**Pull request merged**